tasks/pre_commit.py

```diff
@@ -5,13 +5,25 @@
 
 from tasks.libs.common.color import color_message
 
+DEFAULT_PRE_COMMIT_CONFIG = ".pre-commit-config.yaml"
+DEVAGENT_PRE_COMMIT_CONFIG = ".pre-commit-config-devagent.yaml"
+
+
+def update_pyapp_file() -> str:
+    with open(DEFAULT_PRE_COMMIT_CONFIG) as file:
+        data = file.read()
+        data = data.replace("entry: 'inv", "entry: 'devagent")
```

Comment:
I think you should match `'inv ` or `'invoke ` if we use invoke instead

Reply:
addressed in https://github.com/DataDog/datadog-agent/pull/25910/commits/3703b2605e77258e4c7ac693688208e4c527cb75